MAINTAINERS.md

```diff
@@ -0,0 +1,79 @@
+# What do maintainers do?
+
+Maintainers are people who take an active role in writing code that advances the Baseline Protocol and/or related projects (e.g. the Radish34 Demo). They are primarily responsible for:
+ - Contributing code to the project in the form of PRs that are linked to open and prioritized issues 
+ - Reviewing and merging PRs into the master branch
+ - Cutting, testing, and releasing new versions of the related Baseline projects
+ - Working with the TSC and SSC to advance the Baseline Protocol
+
+ They can/should also contribute in the following ways:
+ - Writing epics and issues to guide development
+ - Setting up and supporting infrastructure (running demos, CI systems, community projects, etc...) that further Baseline
+ - Working with the community to help with adoption
+ - Presenting the project and key technologies to the public (in-person, webinar, videos, articles, etc...)
+ 
+
+# Who are the current maintainers?
+
+Code and Systems maintainers
+  - [Sam Stokes](https://github.com/bitwiseguy) 
+  - [Patrick Macom](https://github.com/pmacom)
+  - [Kartheek Solipuram](https://github.com/skarred14)
+  - [Brian Chamberlain](https://github.com/breakpointer)
+  - [Kyle Thomas](https://github.com/kthomas)
+  - [Hudson Jameson](https://github.com/souptacular)
+
+Documentation maintainer/Technical Product Director
+
+  - [John](https://github.com/humbitious)
+
+
+# How to become a maintainer?
+
+There are two ways to become a maintainer: You are asked by a current maintainer, or you make request to an existing maintainer to become one.
+
+With either path you become a "provisional maintainer". As such you will need to show consistent contributions of code to the project. This can be in the form of pull requests that get merged into master. Or it can be in the form of system architecture and related artifacts that guide the development activities of others. 
+
+All provisional maintainers must meet with the existing maintainers and demonstrate they are capable of the following: 
+ - Running the project locally
+ - Using the testing framework
+ - Explaining the components of the system architecture
+ - Walking through the code and explain the baseline process
+
+Once the provisional maintainer demonstrates their capabilities they existing maintainers will vote during the next scheduled maintainer meeting to give the prospect full maintainer status. Members must vote with 2/3rds majority to add a new maintainer. Voting that results in a tie or potentially other issue will be brought to the TSC for review.
```

Comment:
"demonstrates their capabilities they existing" --> "demonstrates their capabilities, the existing"

Reply:
Fixed